src/components/MessageHTMLBody.vue

```diff
@@ -114,6 +115,11 @@ export default {
 			iframeDoc
 				.querySelectorAll('[data-original-style]')
 				.forEach((node) => node.setAttribute('style', node.getAttribute('data-original-style')))
+			iframeDoc
+				.querySelectorAll('style[data-original-content]')
+				.forEach((node) => {
+					node.innerHTML = node.getAttribute('data-original-content')
```

Comment:
I haven't tested this, but could I use this to bypass any other kind of sanitization we may be running on the HTML?

As in: what if someone sends an email that has something like:

```
<span data-original-content="<img src=&quot;doesnotexist&quot; onerror=&quot;alert(1)&quot;">
```

Or don't we have any kind of sanitization anymore regardless as we sandbox the iframe?

Reply:
The selector only selects `style` elements with the attribute `data-original-content` so your example won't have any effect. 

Nevertheless, an attacker could send a forged style tag containing some malicious content but our sanitation extracts, merges and then sanitizes all style sheets at once so theoretically this shouldn't be exploitable. The rendered html email will only have one `style` element right at the beginning.